caas/kubernetes/provider/application/application.go

```diff
@@ -208,6 +208,40 @@ func (a *app) Ensure(config caas.ApplicationConfig) (err error) {
 	}
 	applier.Apply(&roleBinding)
 
+	clusterRole := resources.ClusterRole{
+		ClusterRole: rbacv1.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:        a.qualifiedClusterName(),
+				Labels:      a.labels(),
+				Annotations: a.annotations(config),
+			},
+			Rules: defaultApplicationClusterRules,
+		},
+	}
+	applier.Apply(&clusterRole)
+
+	clusterRoleBinding := resources.ClusterRoleBinding{
+		ClusterRoleBinding: rbacv1.ClusterRoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:        a.qualifiedClusterName(),
+				Labels:      a.labels(),
+				Annotations: a.annotations(config),
+			},
+			RoleRef: rbacv1.RoleRef{
+				Name: a.serviceAccountName(),
```

Comment:
Shouldn't this be referencing the cluster role name above?

Reply:
yup, thanks.